shared/teams/team/row-types.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+import * as Types from '../../constants/types/teams'
+type HeaderRow = {
+  type: 'header',
+  key: 'headerKey',
+  teamname: Types.Teamname,
+}
+
+type TabsRow = {
+  type: 'tabs',
+  key: 'tabs',
+  admin: boolean,
+  memberCount: number,
+  teamname: Types.Teamname,
+  newTeamRequests: Array<Types.Teamname>,
+  numInvites: number,
+  numRequests: number,
+  numSubteams: number,
+  loading?: boolean,
+  selectedTab?: string,
+  setSelectedTab: (?Types.TabKey) => void,
+  yourOperations: Types.TeamOperations,
+}
+
+/* Member row */
+type MemberRow = {
+  type: 'member',
+  active: boolean,
+  fullName: string,
+  username: string,
+  teamname: string,
+  key: string,
+}
+
+/* Subteams rows */
+type SubteamIntroRow = {
+  type: 'subteam',
+  subtype: 'intro',
```

Comment:
`type` is for delegating in main row renderer, `subtype` is for the body row renderers